win32ss/gdi/ntgdi/coord.h

```diff
@@ -131,6 +131,9 @@ DC_vXformDeviceToWorld(
     PMATRIX pmx;
 
     pmx = DC_pmxDeviceToWorld(pdc);
+    if (!MX_IsValid(pmx))
+        return;
```

Comment:
What does pptlDest contain if this check fails? (-> invalid value?)

Reply:
I have no idea.

Reply:
Maybe it needs a testcase of `NtGdiTransformPoints`.

Reply:
I added a testcase of `NtGdiTransformPoints` function in #1542.